manifest/je.dj/GradientDriverEasing/info.json

```diff
@@ -0,0 +1,23 @@
+{
+	"name": "GradientDriverEasing",
+	"id": "je.dj.GradientDriverEasing",
+	"description": "Adds easing function presets to ValueGradientDriver components.",
+	"category": "Inspectors",
+	"sourceLocation": "https://github.com/djsime1/GradientDriverEasing",
```

Comment:
You should probably link to the `https://github.com/art0007i/CustomUILib/releases/download/2.0.0/CustomUILib.dll` file (or latest release in their repo) in your download steps instead of providing a copy with your releases